.github/workflows/pull-request.yml

```diff
@@ -52,18 +83,57 @@ jobs:
         with:
           npm-auth-token: ${{ secrets.NPM_TOKEN }}
 
+      - name: Cache Playwright browsers
+        if: ${{ success() || failure() }}
```

Comment:
Why do we check for success or failure and we execute it in both cases?

Reply:
Maybe you right we can remove it for cache. This is required only for Test Steps so they will be marked as Failed but continue to next Test without failing whole job immediately